mac.js

```diff
@@ -99,6 +100,12 @@ module.exports = {
         mv(path.dirname(contentsPath), finalAppPath, cb)
       })
 
+	  if (appPlist.CFBundleExecutable !== 'Electron') {
```

Comment:
In what case will this value be `Electron`? (The bigger question I'm asking is, is the conditional necessary?)